data/interfaces/default/welcome.html

```diff
@@ -399,7 +470,19 @@ <h3 style="line-height: 50px;">Welcome!</h3>
             }
         });
     });
+
+    function getServerOptions(token) {
+        /* Set token and returns server options */
+        $.ajax({
+            url: "discover/" + token,
```

Comment:
Curious why you just didn't do this all in javascript and access plex.tv directly?